examples/todos/lib/main.dart

```diff
@@ -261,19 +257,15 @@ class TodoItem extends HookConsumerWidget {
           }
         },
         child: ListTile(
-          onTap: () {
-            itemFocusNode.requestFocus();
-            textFieldFocusNode.requestFocus();
-          },
+          onTap: () => isFocused.value = true,
```

Comment:
Focus is the responsibility of FocusNode

Not using FocusNode isn't reasonable, as it'll break the logic of "focusing one item unfocus the previously focused item"

Reply:
Thank you for the advice, I learned a lot.
I'm going to try to fix it.

Reply:
Making a useIsFocused should be the solution.

Reply:
Yes this is fine

I didn't mean to add a new hook to flutter_hooks, just making one for the example.

Reply:
@davidmartos96 Oh, I learned a lot from that!

Reply:
@davidmartos96 
Hi David, Do you plan to submit the code you fixed for additional PR?
If David doesn't plan to submit a PR, I'll add it to this PR.

Reply:
Thanks, David, I will add that 👍

"I will add it to this PR." might have been a rude English expression.
To be precise, I wanted to tell David "I would like to add to this PR" or "I want to add to this PR."
My English is not very good, I'm still learning.

Reply:
> Thanks, David, I will add that 👍
> 
> "I will add it to this PR." might have been a rude English expression.
> To be precise, I wanted to tell David "I would like to add to this PR" or "I want to add to this PR."
> My English is not very good, I'm still learning.

No worries 😉

Reply:
Hi, Remi,
Updated code to include useIsFocused.
Would you take a look at the updated code?